pep-0225.txt

```diff
@@ -4,13 +4,13 @@ Version: $Revision$
 Last-Modified: $Date$
 Author: hzhu@users.sourceforge.net (Huaiyu Zhu),
         gregory.lielens@fft.be (Gregory Lielens)
-Status: Rejected
+Status: Superseded
 Type: Standards Track
 Content-Type: text/x-rst
 Created: 19-Sep-2000
 Python-Version: 2.1
 Post-History:
-Resolution: :pep:`PEP 465 -- Rejected alternatives to adding a new operator <465#rejected-alternatives-to-adding-a-new-operator>`
+Superseded-By: 465
```

Comment:
I assume you mean 211, 225

Reply:
I ended up just reverting these changes since its not really the intended use of the `Superseded` status per PEP 1, and instead moved this to an explanatory note at the top of the PEP. I also added a missing `Resolution` link to the aforementioned PEP 465 to the message officially accepting it.